security/pkg/server/ca/server.go

```diff
@@ -101,6 +101,21 @@ func (s *Server) CreateCertificate(ctx context.Context, request *pb.IstioCertifi
 	return response, nil
 }
 
+// rootCertExpirationSeconds returns the unix timestamp when the root becomes expires.
+func rootCertExpirationSeconds(ca ca.CertificateAuthority) float64 {
```

Comment:
Should we change it to rootCertExpirationTime? the current one seems to be the number of seconds left.

Reply:
Done, changed the metric name to rootCertExpiryTimestamp

Reply:
It's still "rootCertExpirationSeconds"